src/Symfony/Component/Console/Output/SyncOutputDecorator.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Output;
+
+use Symfony\Component\Console\Formatter\OutputFormatterInterface;
+
+/**
+ * @author Vadim Zharkov <hushker@gmail.com>
+ * Synchronize one output with another
```

Comment:
Please move the description above the author and add a blank line in between

Reply:
thx, fixed